cypress.config.mjs

```diff
@@ -1,11 +1,14 @@
 import { defineConfig } from 'cypress'
```

Comment:
I'd like to keep typescript here please

Reply:
I'd like to keep typescript here please.
I see the `esm` import issue in the commit comment, but I fail to see how that's relevant to this PR :thinking: 
We might not even need to specify the `__dirname` at all :)

Reply:
> Why the rename from ts to mjs?

Short:
Because I made the translations a module.

Long:
Because `ts` is the same as the package type, so for the package it is commonjs Typescript. Every `import` is a `require`. Not sure if Cypress or webpack is the problem but even when changing the `webpackConfig` part to dynamically import `translations.mjs` it leads to.
> Error [ERR_REQUIRE_ESM]: require() of ES Module nextcloud-vue/resources/translations.mjs not supported.

Moreover Cypress supports `mjs` but not `mts`.
So either revert making the `resources/translations` helper a module or make the cypress config a module js file.

Reply:
But yes that is unrelated to this PR and I will revert that change.

Reply:
Done, but it is not that easy see the commit message of the second commit.